packager/media/base/macros.h

```diff
@@ -46,9 +46,7 @@
 #endif
 
 #ifndef FALLTHROUGH_INTENDED
-#define FALLTHROUGH_INTENDED \
-  do {                       \
-  } while (0)
+#define FALLTHROUGH_INTENDED
```

Comment:
What is the reason of removing the original definition?

Reply:
In an earlier PR, you advised me to remove the do/while(0) around some other macro definition.  So I did the same here.  I don't recall it causing any issues.